src/services/classifier.ts

```diff
@@ -814,7 +814,7 @@ namespace ts {
          * False will mean that node is not classified and traverse routine should recurse into node contents.
          */
         function tryClassifyNode(node: Node): boolean {
-            if (isJSDocNode(node)) {
+            if (isJSDoc(node)) {

```

Comment:
This means we will now classify inner JSDoc nodes -- just not the top-level node which contains comment text, and tags (which can also contain comment text) are handled as part of that rather than as part of this code path.